packages/components/.storybook/head.html

```diff
@@ -16,4 +16,8 @@
     .tc-list-display-table tr > td + td {
         max-width: 400px;
     }
+
+    .list-container {
+        height: 80vh;
```

Comment:
I'm ok to keep a height so we will be able to tests scroll. The list may be constrained by it's container.
If you put height you will not be able to test scroll